gradle.properties

```diff
@@ -9,7 +9,7 @@ kotlin_plugin_version=213-1.7.0-RC2-release-258-IJ6777.52
 jsoup_version=1.14.3
 idea_version=213.6777.52
 language_version=1.4
-jackson_version=2.13.3
+jackson_version=2.12.7
```

Comment:
Can you add a comment to prevent this from updating?

Reply:
It's already fixed and is expected to be included in the next version, which is `2.13.4`, but release date is unclear.

So it'll most likely be fixed the next time we update it